extensions/wikia/CommunityPage/templates/topContributors.mustache

```diff
@@ -19,7 +19,7 @@
 				<a href="{{profilePage}}">{{{avatar}}}</a>
 			</div>
 			<div class="contributor-details">
-				{{count}}. <a href="{{profilePage}}">{{userName}}</a>
+				{{count}}. <a data-tracking="top-contributors" href="{{profilePage}}">{{userName}}</a>
```

Comment:
@kamilkoterba Right now, the special tracking has the tag -modal added, with no suffix for the main module. I'm ok with adding -module to the ones in the main module. I'm not so sure about adding -user to the end of everything.